Bottom: 67d12d5c3c600a05d60992a81b960f5724af81a8
Top:    5fe583499abc53626c64a2155431814621e96499
Author: Mark de Wever <koraq@xs4all.nl>
Date:   2021-08-14 17:30:01 +0200

Refresh of implement-the-basic-arithmetic

---

diff --git a/utils/check_coverage.py b/utils/check_coverage.py
index 233f7d8..e5e1eb6 100755
--- a/utils/check_coverage.py
+++ b/utils/check_coverage.py
@@ -32,6 +32,12 @@ not_covered = {
             'branches' : 1,
             'regions': 2,
         },
+    path + '/src/modules/calculator/model.cpp' :
+        {
+            'lines' : 0,
+            'branches' : 3,
+            'regions': 0,
+        },
 }
 
 file = open(path + '/build/ccov/tests.json')
